### PR TITLE
Update bundle description and image

### PR DESCRIPTION
**Pull request by thejasn**
Fixes bundle description and image visible on operator hub.

### CODE REVIEW

**Comment by thejasn** [comment]
/label docs-approved
/label px-approved
/label-qe-approved
This fixes CPaaS related configs on operator bundle.

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -42,11 +58,13 @@ spec:
       kind: CertManager
       name: certmanagers.operator.openshift.io
       version: v1alpha1
-  description: cert-manager-operator
-  displayName: cert-manager-operator
+  description: |
+    The cert-manager Operator for Red Hat OpenShift provides seamless support for cert-manager, which automates certificate management.
+    For more information, see [cert-manager.io](https://cert-manager.io).  displayName: cert-manager-operator
```
Comment (xingxingxia) [inline config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:63]:
```suggestion
    For more information, see [cert-manager.io](https://cert-manager.io).
```
Forgot removing trailing displayName: cert-manager-operator?

Comment (xingxingxia) [reply config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:63]: /hold
Adding /hold. Feel free to unhold if above comment is not right.

**Comment by swghosh** [comment]
/close
We've moved the changes in this PR to #89.

**Pull request closed without merging**